plugins/baseplugin/src/main/resources/META-INF/cxs/rules/evaluateProfileSegments.json

```diff
@@ -2,6 +2,7 @@
     "metadata" : {
         "id": "_t7caz8npj_evaluateProfileSegments",
         "name": "Evaluate segments",
+        "enabled": false,
```

Comment:
Will this disable it for Unomi profiles as well?

Reply:
Yeah but this is taken care by our `evaluate_segments_write_to_upapi_rules` [rule](https://github.com/Reflektion/rfk-data-unomi-config/blob/DA-release-2023/scripts/rules/evaluate_segments_write_to_upapi_rules.json) which performs this action internally. Hence disabling it here so that it won't be called twice